packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -265,6 +265,11 @@ export default function vercelServerless({
 						ssr: {
 							external: ['@vercel/nft'],
 						},
+						build: {
+							rollupOptions: {
+								external: ['astro/env/setup'],
+							},
+						},
```

Comment:
Questions:
1. Could we put this in `ssr.external`? (I think they work the same)
2. Could we only add this if `astro/env/setup` exists? We can try importing it here, then decide adding the config.